From de1f047987a24beb866edaa1e826a9446f62fc24 Mon Sep 17 00:00:00 2001
From: Mao Minkai <maominkai@wxiat.com>
Date: Thu, 11 Aug 2022 09:57:38 +0800
Subject: [PATCH] sw64: make RO_DATA PAGE_SIZE aligned
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: de1f047987a24beb866edaa1e826a9446f62fc24
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Signed-off-by: Mao Minkai <maominkai@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/kernel/vmlinux.lds.S | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/sw_64/kernel/vmlinux.lds.S b/arch/sw_64/kernel/vmlinux.lds.S
index a106be42121f..07bc3d8ee7e4 100644
--- a/arch/sw_64/kernel/vmlinux.lds.S
+++ b/arch/sw_64/kernel/vmlinux.lds.S
@@ -33,7 +33,7 @@ SECTIONS
 	} :text
 	_etext = .;	/* End of text section */
 
-	RO_DATA(4096)
+	RO_DATA(PAGE_SIZE)
 
 	/* Will be freed after init */
 	__init_begin = ALIGN(PAGE_SIZE);
-- 
2.33.0

